e2e-tests/web/browserstack/wdio-browserstack-chrome.conf.js

```diff
@@ -0,0 +1,7 @@
+const defaultConfig = require('./wdio-browserstack.conf')
+
+const config = Object.assign({}, defaultConfig.config)
+config.capabilities = config.capabilities.filter(cap =>
+    cap.browserName.toLowerCase() === 'chrome'
```

Comment:
This is probably done best with args passed to the config. But for an initial setting I think this is enough